### ISSUE TITLE
Enlarged amygdala

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6730]
**Preferred term label:**
Enlarged amygdala

**Synonyms**
Enlargement of the amygdala

**Definition (free text, please give PubMed ID)**
A increase in the volume (size) of the amygdyla.

Comment: Amygdalar enlargement (AE) is observed in a minority of people with intractable temporal lobe epilepsy.

PMID: 33044668, 21047879, 12933928

**Parent term (use hpo.jax.org/app)**
Abnormal morphology of the limbic system HP:0007343.
Consider an intermediate term: Abnormal morphology of the amygdala with the children:
Enlarged amygdala (above) and Reduced amygdala volume HP:0025444 (exists)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Temporal lobe epilepsy.

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
done

**Comment by DLewisSmith** [comment]
@pnrobinson , are you able to include "Abnormal morphology of the amygdala" as well? Thank you,
